### PR TITLE
Add dao to playground

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Description

Added Dao page in playground using Vue 3 and updated UI

### Changes

A dao view in portal in playground

### Related Issues

#1019 

### Checklist

- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/playground/src/portal/dao_view.vue [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+
+  if (grid) {
+    try {
+      await grid.dao.vote({
+        address: profile.value.address,
+        farmId: selectedFarm.value,
+        approve: castedVote.value,
+        hash: selectedProposal.value,
+      });
+      createToast("Transaction Complete!", {
```
Comment (AhmedHanafy725) [inline packages/playground/src/portal/dao_view.vue:316]: let's use more meaningful messages and add a leading icon

**Comment by AhmedHanafy725** [comment]
the description's value shouldn't be bold and the date as well

image

**Comment by AhmedHanafy725** [comment]
when trying to vote, the background is white. it should be transparent
image

**Comment by AhmedHanafy725** [comment]
add some spacing before the info icon
image

**Comment by amiraabouhadid** [comment]
background color to vote dialog isnt completely white in my browser it looks like this
Screenshot from 2023-09-27 12-22-40

which is consistent with other dialogs in playground for instance: 
Screenshot from 2023-09-27 12-24-49

File: packages/playground/src/portal/dao_view.vue [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+      await grid.dao.vote({
+        address: profile.value.address,
+        farmId: selectedFarm.value,
+        approve: castedVote.value,
+        hash: selectedProposal.value,
+      });
+      createToast("Voted for proposal!", {
+        position: "top-right",
+        hideProgressBar: true,
+        toastBackgroundColor: "green",
```
Comment (zaelgohary) [inline packages/playground/src/portal/dao_view.vue:314]: Can we change the toast background color?

File: packages/playground/src/portal/dao_view.vue [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+        timeout: 5000,
+        showIcon: true,
+      });
+    } catch (err) {
+      createToast(`Vote Failed!`, {
+        position: "top-right",
+        hideProgressBar: true,
+        toastBackgroundColor: "red",
```
Comment (zaelgohary) [inline packages/playground/src/portal/dao_view.vue:322]: Also the color here needs to be changed.
image

File: packages/playground/src/portal/dao_view.vue [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+
+    <div v-else-if="proposals?.active?.length == 0 && proposals?.inactive?.length == 0">
+      <v-card class="my-3 pa-3 d-flex justify-center">
+        <h3>No proposals at this time</h3>
+      </v-card>
+    </div>
+    <div v-else>
+      <v-card>
+        <h4 class="d-flex justify-center pa-4">
+          User, you can now vote on proposals!
```
Comment (zaelgohary) [inline packages/playground/src/portal/dao_view.vue:19]: Can we change the phrasing here?

File: packages/playground/src/portal/dao_view.vue [hunk 5]
```diff
@@ -0,0 +1,338 @@
...
+              v-model="searchTerm"
+              variant="underlined"
+              label="Search by proposal description"
+              class="pa-3"
+            ></v-text-field>
+          </v-card>
+
+          <v-card class="my-3 pa-3" v-for="(proposal, i) in filteredProposals(tab.content.value)" :key="i">
+            <v-card-title class="pa-3 mb-2" v-if="proposal.action">
+              {{ proposal.action.toUpperCase() }}
```
Comment (zaelgohary) [inline packages/playground/src/portal/dao_view.vue:40]: No need to make the title uppercased.

File: packages/playground/src/portal/dao_view.vue [hunk 6]
```diff
@@ -0,0 +1,338 @@
+<template>
+  <v-container fluid>
+    <v-card color="#1AA18F" class="my-3 pa-3 text-center">
```
Comment (zaelgohary) [inline packages/playground/src/portal/dao_view.vue:3]: The card should be display flex.

**Review by zaelgohary — CHANGES_REQUESTED** [review]

File: packages/playground/src/portal/dao_view.vue (hunk 4 above)
Comment (amiraabouhadid) [reply packages/playground/src/portal/dao_view.vue:19]: It is the same as given in figma [design ](https://www.figma.com/proto/KAQ6497YfczErqfh1Bz62l/Dashboard2?type=design&node-id=54-2667&t=zNC6x0dPbosoH4bW-0&scaling=scale-down&page-id=0%3A1&starting-point-node-id=1%3A1248)

**Review by zaelgohary — APPROVED** [review]

**Pull request merged**